### DISCUSSION TITLE
Intercept links computing from extension

### DISCUSSION THREAD

**Discussion by vvavrychuk** [microsoft/vscode-discussions#301] [Extension Development QnA] [answered]
I am looking how to fix issue [#86741](https://github.com/microsoft/vscode/issues/86741) with Asciidoc links incorrectly highlighted
![image](https://user-images.githubusercontent.com/152563/206867114-10942bc4-a460-42ef-984e-f77056bb3b99.png)

I have found that links are parsed in [computeLinks](https://github.com/microsoft/vscode/blob/f63eaa61fb7e4599824574add2e29ae5e677cf77/src/vs/editor/common/languages/linkComputer.ts#L213). But, this function is language independent. So, changing it globally will lead to `[]` ignored when it is part of link in non-Asciidoc documents.

Is there a way to provide document links from extension?

**Comment by isidorn** [comment, marked as answer]
No, this is currently not possible from extensions.
